webapp/src/Controller/Jury/ExecutableController.php

```diff
@@ -59,12 +59,25 @@ public function indexAction(Request $request): Response
             'icon' => ['title' => 'type', 'sort' => false],
             'execid' => ['title' => 'ID', 'sort' => true,],
             'type' => ['title' => 'type', 'sort' => true,],
+            'badges' => ['title' => 'problems', 'sort' => false],
             'description' => ['title' => 'description', 'sort' => true,],
         ];
 
         $propertyAccessor  = PropertyAccess::createPropertyAccessor();
         $executables_table = [];
         foreach ($executables as $e) {
+            $badges = [];
+            if ($this->dj->getCurrentContest()) {
+                foreach([$e->getProblemsCompare(), $e->getProblemsRun()] as $problemList) {
```

Comment:
If you use `array_merge` here, you can reduce the two outer loops to a single one

Reply:
Tried to fix this in the query to speed it up further.